src/components/navbar/navbar.ts

```diff
@@ -92,116 +122,54 @@ export class MNavbar extends BaseNavbar implements MNavbarInterface {
             parseInt(elComputedStyle.paddingRight, 10) - parseInt(elComputedStyle.borderLeftWidth, 10) -
             parseInt(elComputedStyle.borderRightWidth, 10);
         if (width < listEl.clientWidth) {
-            let height: number = listEl.clientHeight;
+            this.computedHeight = listEl.clientHeight;
             this.hasScrolllH = true;
-            wrapEl.style.height = height + 40 + 'px';
-            this.$el.style.height = height + 'px';
-            this.scrollToSelectedElem();
+            wrapEl.style.height = this.computedHeight + 40 + 'px';
+            this.$el.style.height = this.computedHeight + 'px';
         } else {
             this.hasScrolllH = false;
             wrapEl.style.removeProperty('height');
             this.$el.style.removeProperty('height');
         }
     }
 
-    private scrollLeft(event: MouseEvent): void {
-        let wrapEl: HTMLElement = this.$refs.wrap as HTMLElement;
-        wrapEl.scrollLeft = wrapEl.scrollLeft - ((this.$el.clientWidth - 88) / 2.5);
-    }
-
-    private scrollRight(event: MouseEvent): void {
-        let wrapEl: HTMLElement = this.$refs.wrap as HTMLElement;
-        wrapEl.scrollLeft = wrapEl.scrollLeft + ((this.$el.clientWidth - 88) / 2.5);
-    }
-
-    public get model(): string {
-        return this.internalValue;
+    private get ComputedHeight(): string {
+        return this.computedHeight + 'px';
     }
 
-    public set model(value: string) {
-        this.internalValue = this.disabled ? '' : value;
+    private get buttonSkin(): string {
+        return this.skin == 'dark' ? this.skin : 'light';
     }
 
-    private setItem(): void {
-        this.$children.forEach((child, index, arr) => {
-            if (index == 0 && arr.length >= 1) {
-                child['isFirst'] = true;
-            }
-            if (arr.length - 1 === index && arr.length > 1) {
-                child['isLast'] = true;
-            }
-        });
-    }
-
-    private initLine(): void {
-        this.$children.forEach((child, index, arr) => {
-            if (child.$props.value == this.value) {
-                if (this.skin == MNavbarSkin.Light) {
-                    this.setLinePosition(child.$el);
-                }
-                if (this.skin == MNavbarSkin.Arrow) {
-                    this.setArrowPosition(child.$el);
-                }
-            }
-        });
+    private get isLightSkin(): boolean {
+        return this.skin == 'light' ? true : false;
```

Comment:
`return this.skin == 'light' ? true : false;`
se remplace par
`return this.skin == 'light';`